### PR TITLE
Changed F1 from OOC to ahelp

### PR DESCRIPTION
**Pull request by danhibiki**
This was put off for far too long by our previous team

### CODE REVIEW

File: html/changelog.html [hunk 1]
```diff
@@ -58,10 +58,21 @@
 <!-- DO NOT REMOVE OR MOVE THIS COMMENT! THIS MUST BE THE LAST NON-EMPTY LINE BEFORE THE LOGJS #ADDTOCHANGELOGMARKER# -->
 
 <div class='commit sansserif'>
+	<h2 class='date'>25 April 2015</h2>
+	<h3 class='author'> updated:</h3>
```
Comment (AliiTea) [inline html/changelog.html:62]: Also you are missing your name here :^)

**Comment by danhibiki** [comment]
I'm very aware. I was more focused on making Jamie's changelog addition less of a burn on the previous coding team for hippie and more of an actual documentation of his additions to the codebase.

**Comment by j-mie** [comment]
[19:45:31] Dan Hibiki: Its in our current changelog
[19:45:39] Dan Hibiki: "I leave for a month and you guys break everything"

---

Little intention in getting caught up in the drama around this, but you realise I was referring to the fact that I'd left Hippie and were focusing on other things. Atlas came around after I left the Skype chats, so when I made that changelog I was referencing to the fact that I'd left, and in the space of a month people had broken alot of things.

**Comment by SplergeBucket** [comment]
And all of those people who "had broken alot of things" are on your new codebase, with none of them currently here anymore. 
:+1:  :8ball:  :+1:

**Pull request closed without merging**